dash_application.py

```diff
@@ -115,15 +116,20 @@ def SelectionToObject(x):
     ]),
 
      # Network plot
-    dcc.Graph(id='network_plot')   
+    dcc.Graph(id='network_plot'), 
+
+    # Data table
+    dash_table.DataTable(id='table')
 
 ])
 #-------- Callback --------
 
 @app.callback(
     [Output(component_id='imports_between_two_countries_value', component_property='figure'),
      Output(component_id='imports_between_two_countries_kg', component_property='figure'),
-     Output(component_id='network_plot', component_property='figure')],
+     Output(component_id='network_plot', component_property='figure'),
```

Comment:
Add  component for a dash data table filtered at the specific country - but this is not filtering for the specific exported country